multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -433,7 +433,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Shayami", "https://shayami.com", "es"),
         SingleLang("Shiba Manga", "https://shibamanga.com", "en"),
         SingleLang("Shield Manga", "https://shieldmanga.io", "en", overrideVersionCode = 3),
-        SingleLang("Shinigami", "https://shinigami.moe", "id", overrideVersionCode = 10),
+        SingleLang("Shinigami", "https://shinigamitoon.com/", "id", overrideVersionCode = 11),
```

Comment:
```suggestion
        SingleLang("Shinigami", "https://shinigamitoon.com", "id", overrideVersionCode = 11),
```
Remove trailing slash